src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1908,8 +1909,8 @@ namespace ReportSizingManager {
                     } else {
                         AutosizeDes = FinalZoneSizing(CurZoneEqNum).CoolDesHumRat;
                     }
-                    if (AutosizeDes > DataDesInletAirHumRat && DataCapacityUsedForSizing > 0.0 &&
-                        UtilityRoutines::SameString(CompType, "COIL:COOLING:WATER")) { // zone coil uses mDot>0, sys coil uses cap > 0
+                    if (AutosizeDes > DataDesInletAirHumRat && (UtilityRoutines::SameString(CompType, "COIL:COOLING:WATER") ||
+                                                                UtilityRoutines::SameString(CompType, "COIL:COOLING:WATER:DETAILEDGEOMETRY"))) {
```

Comment:
Adding another string comparison seems counterproductive.  I wouldn't hold everything up, but it seems like we should be passing int's around, not strings, if we're going to be doing comparisons in here.

Reply:
We'll integerize these on the inevitable sizing consolidation.